daemon/cluster/cluster.go

```diff
@@ -276,21 +284,31 @@ func (c *Cluster) startNewNode(forceNewCluster bool, localAddr, remoteAddr, list
 		}
 	}
 
+	specAvailability := swarmapi.NodeAvailabilityActive
```

Comment:
By default, this should be an empty string. We should only send an availability value if the user specifically chooses one with the `--availability` flag.